### PR TITLE
[SIM_FLUTTER-95] [Integrate] Dynamically change navbar

### PR DESCRIPTION
**Pull request by jstephend-sun**
### Issue links
- [backlog](https://framgiaph.backlog.com/view/SIM_FLUTTER-95)
- [slack](https://sun-philippine.slack.com/archives/C059L7XAXFB/p1688028647145679)

### Definition of done
- [x] change navbar based on user's is_admin value

### Recording
- admin

image

- user

image

### CODE REVIEW

**Comment by Dbulawan13** [comment]
<RESOLVED: NOT A BUG>
Comment:

- [ ] Unable to log in. You will be redirected to Visual Studio Code (VSCode).

https://github.com/framgia/sph-flutter/assets/86758474/255f5019-abd4-45bf-b491-84ba5866909d

Comment: 1

**Review by Dbulawan13 — DISMISSED** [review]
Approve PR #36 

Working:
- Display correct bottom navbar
  - if admin user is logged in
  - if user is logged in
- Abe to login
- Able to logout

https://github.com/framgia/sph-flutter/assets/86758474/c1f267b6-07d3-449f-988e-907aa3675094

Testing Horus: 1
Comment: 0
Iteration: 0

File: frontend/lib/src/controllers/home_screen_controller.dart [hunk 1]
```diff
@@ -1,16 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:frontend/src/helper/storage.dart';
+import 'package:frontend/src/navigators/dashboard_screen_navigator.dart';
+import 'package:frontend/src/navigators/profile_screen_navigator.dart';
+import 'package:frontend/src/navigators/users_list_screen_navigator.dart';
```
Comment (rdelrosariojr) [inline frontend/lib/src/controllers/home_screen_controller.dart:9]: sort the imports

**Review by rdelrosariojr — COMMENTED** [review]
one minor comment

**Review by rdelrosariojr — APPROVED** [review]
**APPROVED PR36**

reviewed time: 0.28
comments: 1

**Pull request merged**